TrajectoryPlannerROS:
  
  #Robot parameters
  acc_lim_x: 2.5
  acc_lim_y: 0.0
  acc_lim_theta: 1.0

  max_vel_x : 2.0
  escape_vel: -2.0
  max_vel_theta : 1.0
  min_vel_theta : -1.0
  min_in_place_vel_theta: 1.0
  
  holonomic_robot: false

  #Goal parameters
  yaw_goal_tolerance: 0.087 #0.087
  xy_goal_tolerance: 0.1 #0.1

  #Trajectory Scoring Parameters
  # path_distance_bias: 2.8 #default 0.6
  # goal_distance_bias: 0.8 #default 0.8
  occdist_scale : 0.03 #default 0.01

  #Forward Simualtion
  sim_time: 1.5
  vx_samples: 10
  vtheta_samples: 21